english/best_cookies/cookie_bars.md

```diff
@@ -0,0 +1,42 @@
+The following is the best cookie bar recipe
+
+If you didn't like it, you can got back to [backing cookies] (../bake-cookies/cookies.md)
+
+Raspberry "Rhubars"
+
+Ingredients
+
+CRUST 
+2 cups all-purpose flour
+1/2 cup powdered sugar
+1/2 teaspoon baking soda
+1/4 teaspoon table salt 
+3/4 cup cold butter, cubed
+3/4 cup toasted slivered almonds, coarsely chopped 
+Vegetable cooking spray 
+
+RHUBARB-RASPBERRY FILLING
+3/4 cup granulated sugar
+1/4 cup cornstarch
+1/4 teaspoon table salt
+1 pound rhubarb or 1 (1-lb.) package frozen rhubarb, thawed, cut into 1/2-inch-thick slices
+2 (6-oz.) containers fresh raspberries 
+1 teaspoon vanilla extract 
+
+CREAM CHEESE BATTER 
+2 (8-oz.) packages cream cheese, softened
+3/4 cup granulated sugar
+2 large eggs
+1 teaspoon lemon zest
+1 tablespoon fresh lemon juice
+Garnish: powdered sugar
+
+Preparation
+
+1. Prepare Crust: Preheat oven to 350°. Stir together flour and next 3 ingredients in a large bowl; cut butter into flour mixture with a pastry blender until crumbly. Stir in almonds. Press mixture onto bottom of a lightly greased (with cooking spray) 13- x 9-inch pan. Bake 15 to 20 minutes or until lightly browned. Cool on a wire rack until ready to use.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.